src/seedu/addressbook/AddressBook.java

```diff
@@ -5,19 +5,10 @@
  * ====================================================================
  */
 
-import java.io.File;
-import java.io.FileNotFoundException;
-import java.io.IOException;
+import java.io.*;
```

Comment:
Let say you smelly code liek this written y someone. Do you know how to import the right Classes in a jiffy using eclipse?